README.md

```diff
@@ -44,6 +44,8 @@ values in [kube/](kube/) example files.
 * `CLOUD_AWS_S3_REGION` - Cloud AWS S3 region for repository-s3 plugin. See https://www.elastic.co/guide/en/elasticsearch/plugins/5.5/repository-s3.html
 * `XPACK_SECURITY_ENABLE` - Whether X-Pack security plugin is enabled. Default: `false`
 * `XPACK_SECURITY_AUDIT_ENABLE` - Whether to enable auditing to keep track of attempted and successful interactions with Elasticsearch cluster. Default: `false`.
+* `XPACK_AUDIT_INDEX_EVENTS_EXCLUDE` - Excludes the specified auditing events from indexing. By default, no events are excluded. See https://www.elastic.co/guide/en/x-pack/5.5/auditing.html#audit-event-types
```

Comment:
Worth giving an example of how multiple fields can be excluded i.e. comma separated list of values